lib/experimental/Widgets/Widget/index.tsx

```diff
@@ -141,7 +141,7 @@ const Container = forwardRef<
       </CardContent>
       {action && (
         <CardFooter>
-          <Button variant="outline" size="md" {...action} />
+          <Button variant="neutral" size="md" {...action} />
```

Comment:
The size used in the design is the small one (`sm`).

Reply:
Thanks! forgot about the size